v3.0-RC2/geofencing_zones.json

```
@@ -0,0 +1,202 @@
+                            },
+                            "ride_through_allowed": {
+                              "description": "Is the ride allowed to travel through this zone?",
+                              "type": "boolean"
+                            },
+                            "maximum_speed_kph": {
+                              "description": "What is the maximum speed allowed, in kilometers per hour?",
+                              "type": "integer",
+                              "minimum": 0
+                            },
+                            "station_parking": {
+                              "description": "Vehicle MUST be parked at stations defined in station_information.json within this geofence zone",
+                              "type": "boolean"
+                            }
+                          },
+                          "required": [
+                            "ride_start_allowed",
+                            "ride_end_allowed",
+                            "ride_through_allowed"
+                          ]
+                        }
+                      }
+                    }
+                  },
+                  "geometry": {
+                    "description": "A polygon that describes where rides might not be able to start, end, go through, or have other limitations. Must follow the right-hand rule.",
+                    "title": "GeoJSON MultiPolygon",
+                    "type": "object",
+                    "properties": {
+                      "type": {
+                        "type": "string",
+                        "enum": ["MultiPolygon"]
+                      },
+                      "coordinates": {
+                        "type": "array",
+                        "items": {
+                          "type": "array",
+                          "items": {
+                            "type": "array",
+                            "minItems": 4,
+                            "items": {
+                              "type": "array",
+                              "minItems": 2,
+                              "items": {
+                                "type": "number"
+                              }
+                            }
+                          }
+                        }
+                      }
+                    },
+                    "required": ["type", "coordinates"]
+                  }
+                },
+                "required": ["type", "geometry", "properties"]
+              }
+            }
+          },
+          "required": ["type", "features"]
+        },
+        "global_rules": {
+          "description": "Array of Rule objects defining restrictions that apply globally in all areas as the default restrictions, except where overridden with an explicit geofencing zone.",
+          "type": "array",
+          "items": {
+            "type": "object",
+            "properties": {
+              "vehicle_type_id": {
```

Let's please replace `vehicle_type_id` with `vehicle_type_ids` here as well. Thank you!

```suggestion
              "vehicle_type_ids": {
```